infor-design/enterprise-wc-examples
# 792 - Add more Sveltekit examples, upgrade Sveltekit

**Describe what this PR does:**
This PR improves the Sveltekit demoapp in the folllowing ways:
- Adds examples for IdsAbout, IdsAccordion, IdsActionPanel, IdsActionSheet, IdsAlert, IdsAppMenu, IdsAreaChart, IdsAxisChart, and IdsBadge
- Upgrades Sveltekit to its latest beta release and converts the project structure to accomodate [Sveltekit's recent routing changes](https://github.com/sveltejs/kit/discussions/5774).
- Overall improvements to Typescript types and demoapp structure (trying to harden this so other devs can add to the demoapp predictably).

**Related Issues:**
Related to infor-design/enterprise-wc#792

**How to test:**
- Pull/Build/Run the Sveltekit Demoapp
- Open http://localhost:3000
- Take a look at all component pages, especially for the components listed above.  Ensure they all load and work as expected.
